server.go

```diff
@@ -264,11 +316,9 @@ func (s *Server) listenAndServe() error {
 	s.debugLog("server: gracefully shutting down")
 
 	// 1. Tell amqp we want to shut down by canceling all the consumers.
-	for _, consumerTag := range consumerTags {
-		err = inputCh.Cancel(consumerTag, false)
-		if err != nil {
-			return err
-		}
+	err = s.cancelConsumers(inputCh, consumerTags...)
```

Comment:
Any reason for the expansion instead of sending `[]string`?

Reply:
No reason really. Do you prefer that I change?

Reply:
Since it's expanded here and collected in `cancelConsumers()` I think so. And `startConsumers()` returns a `[]string` so I guess it makes more sense to keep the type as is, makes the call to the function clearer to me as well.